modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/MultipleSelectBase.tsx

```diff
@@ -97,7 +97,6 @@ const MultipleSelectBase = ({
 						>
 							<div className="auto autofit-row-center fit-row">
 								<ClayCheckbox
-									aria-label={item.label}
```

Comment:
The implementation of `ClayCheckbox` by default was built in a way where the input element is a child of the label element, thus creating this relationship between input and label, without the need to point a reference with aria-label.

```
<label>
	<input
		{...otherProps}
		checked={checked}
		className={classNames(
			'custom-control-input',
			className
		)}
		disabled={disabled}
		ref={(r) => {
			inputRef.current = r;
			if (typeof ref === 'function') {
				ref(r);
			}
		}}
		type="checkbox"
	/>

	<span className="custom-control-label">
		{label && (
			<span className="custom-control-label-text">
				{label}
			</span>
		)}
	</span>

	{children}
</label>
```

Reply:
I believe that creating a test that selects this element and checks if it is not implementing a label reference (aria-label) is not an effective test. 

I believe that a test shows its value when it fails by revealing an error. 

If I perform a manual test checking for the non-existence of this property, there is only one way for this test to fail again, which would be to add this line again.

But I would like to know your opinion on this, @rebsilva 😄

Reply:
hi @igor-franca! you could do something similar to [this unit test](https://github.com/liferay/liferay-portal/blob/387d567711c76d628a32704dffacf46e974102ce/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/DatePicker/DatePicker.es.js#L27), but i agree with you that it doesn't make much sense on this case. if the clay is already testing on their side this implementation (which i believe they do) we don't need to implement one; also, it doesn't make sense for this test to be on our side.

Reply:
but just to make sure, i would see why this `aria-label={item.label}` was added in the first place, was it along with the creation of the field? was because of a bug? just to make sure we aren't forgetting something here